tests/tls_e2e/common/openssl_utility.h

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openssl/evp.h>
+#include <openssl/ssl.h>
+#include "utility.h"
+
+#define TLS_SERVER "TLS_TEST_SERVER: "
```

Comment:
Might be helpful to name something like `OPENSSL_TEST_SERVER` (same for the mbedtls_utility.h)

Reply:
Moved it to utility.h file so that it can be used in mbedtls file too.